### PR TITLE
  Hello project changes.

### PR DESCRIPTION
**Pull request by frankfembo**
>In this hello microverse project setup, I added:👍 
   - **Html file boilerplate with elements necessary for this project.**
   - **Css file with code to style the header**
   - **A .gitignore file with test.md and node_modules in it. Html & css Linters, eg; .stylelintrc.json,
        .hintrc, linters.yml as they are without modifying them.**
   - **A test.md file with text to be ignored by git.**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (omarsalem7) [inline .gitignore]: - [x] Please, add `test.md` to `.gitignore` also

File: hello.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<<<<<<< Updated upstream
+.hd {
+=======
+.hdng {
+>>>>>>> Stashed changes
+    justify-content: center;
+    font-family: 'Times New Roman', 'Times New Roman', Times, serif;
+    color: blue;
+}
```
Comment (omarsalem7) [inline hello.css]: - [ ] Please, clean your CSS file and leave only the class you want for the h1 tag like this
```
.hdng {
    justify-content: center;
    font-family: 'Times New Roman', 'Times New Roman', Times, serif;
    color: blue;
}
```

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @frankfembo, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️

- [x] Kindly your Pull Request should have a descriptive title telling you what changes are inside. 
- [x] Also your Pull Request should have a summary describing introduced changes in more detail.
- [x] Kindly modify your readme and use the microverse [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and update it according to your project
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (frankfembo) [reply .gitignore]: Added test.md to .gitignore

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @frankfembo :wave: , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
  :heavy_check_mark: Good use of Github flow

 ## Required Changes ♻️
 
_Check the comments under the review._

- [x] You should not alter or modify the linter setup files and Github action workflows provided by Microverse. They are given to be used `as is` and help develop a code that complies with best practice and other students in Microverse. Therefore, you should use the exact folder names and file name for the `linters.yml` linters Github action workflow and add the contents of the empty `.hintrc` file for the Webhint linter in your project. Please refer [this](https://github.com/microverseinc/linters-config/tree/master/html-css) lesson.

image

- [x] You should a professional ReadMe file based on the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. All sections are mandatory to keep even if you don't currently have content for each of them.

- [x] When you create a pull request, you need to add a pull request title and pull request summary that describe the changes that are included. I see that you added the files changed in the Pull Request title but there is no pull request summary. You could have used `Hello Microverse: Setup` as the Pull Request title and the pull request summary can be something like this. 
> In this pull request, I have:
>  - setup github actions
>  - setup linters locally
>  - created hello.html file
>  - added boilerplate HTML code to hello.html
>  - created hello.css file
>  - added basic styling to hello.css
>  - created .gitignore file and add test.md as a content

Kindly refer this GIF from Microverse.
pull_request_desc

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frankfembo — COMMENTED** [review]
In this hello microverse project setup, I added :

- **Html file boilerplate with elements necessary for this project.**
- **Css file with code to style the header <h1> element**
- **A .gitignore file with test.md and node_modules in it. Html & css Linters, eg; .stylelintrc.json, 
     .hintrc, linters.yml as they are without modifying them.**
- **A test.md file with text to be ignored by git.**

**Comment by frankfembo** [comment]
In this hello microverse project setup, I added :

- Html file boilerplate with elements necessary for this project.
- Css file with code to style the header.
   Elements
- A .gitignore file with test.md and node_modules in it. Html & css Linters, eg; .stylelintrc.json,
- .hintrc, linters.yml as they are without modifying them, and github flow.
- A test.md file with text to be ignored by git.

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @frankfembo , 

Great work on making the changes requested by a previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights

Good use of Github flow ✔️
Good name of the PR ✔️

 ## Required Changes ♻️

**1. Good job for adding a description in your PR.**

I think is better to put your description in the first comment on your PR : 
image

here
image

**2. As said the previous reviewer : you have to rename your file** 

image

by this
image

You have to follow this [steps](https://github.com/microverseinc/linters-config/tree/master/html-css) 

**3. The readme file**

Your readme should stick to this.
You can remove non used section but it should stick to the template

image

Please use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md)







## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by williamrolando88 — CHANGES_REQUESTED** [review]
Hi @frankfembo, 

### Great work on making the changes requested by a previous reviewer 👏 👍 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [x] Please make sure to implement the changes required by previous reviewers, your Readme.md file is still different from the required one
image

- [x] Please make sure to rename linters file as previous reviewers requested 
image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Div685 — CHANGES_REQUESTED** [review]
Hi, **@frankfembo**,  I'm Divyesh your reviewer for this PR :man_technologist: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- :heavy_check_mark: descriptive PR :clamp: 

 ## Required Changes ♻️
- [x] [Required] Please make the required changes asked by the previous reviewer.
- [x] [Required] Please remove `test.md` file as it is ignored by `.gitignore` file.
- [x] [Required] Please Change this as it's required.
linter_Screenshot from 2022-05-29 13-20-46
- [x] [Required] Please It's best practice to learn to use the proper `README` file with all the headers as it's mentioned in the [README template](https://github.com/microverseinc/readme-template)


**_Check the comments under the review._**


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Code review by **Divyesh Patel**

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.)_

------

>**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

>_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @frankfembo,** 

### **Great work on making the changes 👏** 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## **Suggested changes**
_Check the comments under the review._

- Hello and nice job on your first app however, you have missed some required changes that the previous TSE asked for.


1. The `.github/workflows/blank.yml` should be renamed as **.github/workflows/linters.yml**
2.  Your `Readme.md` file should contain details you can copy this [ template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add details to it like :( description about the project, how to copy the repo & start the live server, Author name, etc..).





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @frankfembo   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️

- [ ]  You should just use the provided readme template, and customize it with your information as previous reviewer suggested 

image
Having a good project documentation will make it professional, and it will help any person who will visit it to know how to configure it, and The Ideas of what you've worked on. 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

**Review by dasileker — APPROVED** [review]
### **Hi @frankfembo,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 
**Optional**

- **in your  readme getting started section you can add the repo link + the command line on how to open the app example:  open your terminal and past `https://github.com/frankfembo/hello_microvs.git`, cd hello_microvs, and open it  in VSCode via cmd line type `code .`**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**